docs/user_api/index.rst

```diff
@@ -0,0 +1,44 @@
+.. currentmodule:: geocat.viz
+
+User API
+========
+
+Routines
+--------
+
+GeoCAT-viz Plotting Routines
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. autosummary::
+   :toctree: ./generated/
+   
+   geocat.viz.taylor.TaylorDiagram
+   
+   
+GeoCAT-viz Utility Functions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. autosummary::
+   :toctree: ./generated/ 
+
+   geocat.viz.util.add_lat_lon_ticklabels
+   
+   geocat.viz.util.add_major_minor_ticks
+   
+   geocat.viz.util.set_titles_and_labels
+   
+   geocat.viz.util.set_axes_limits_and_ticks
+   
+   geocat.viz.util.truncate_colormap
+   
+   geocat.viz.util.xr_add_cyclic_longitudes
+   
+   geocat.viz.util.set_map_boundary
+   
+   geocat.viz.util.findLocalExtrema
+   
+   geocat.viz.util.plotCLabels
```

Comment:
I'm not sure why, but all of the API links and documentation looks correct except for plotCLabels and plotELabels. When you look at the Sphinx build of the api page, the first half-ish of the arguments are displaying as the raw code (`:param ax :`, `:param transform;`, etc). I'm not too familiar with how the API is being generated so I can't offer a fix for this, but hopefully it isn't something too crazy to accomplish.

Reply:
@michaelavs @anissa111 I'm not sure how to check this if the normal build method is not working, but it is also failing to build on RTD (which is how I usually check) so I was wondering if y'all had suggestions on how to look into this?